### PR TITLE
PIM-3752: Fixed the hard coded entry Select Job on import/export creation

### PR DESCRIPTION
**Pull request by solivier**
| Q                    | A
| -------------------- | ---
| Bug fix?             | No
| New feature?         | No
| BC breaks?           | No
| CI currently passes? | Yes
| Tests pass?          | Yes
| Scenarios pass?      | Yes
| Checkstyle issues?*  | No
| PMD issues?**        | No
| Changelog updated?   | Yes
| Fixed tickets        | PIM-3752
| DB schema updated?   | No
| Migration script?    | No
| Doc PR               | No

### CODE REVIEW

**Comment by jjanvier** [comment]
changelog for BC then GTM

File: src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ pim_import_export:
       error: "L'import de <strong>%label%</strong> a échoué"
       warning: "L'import de <strong>%label%</strong> est terminé mais il y a des avertissements"
       success: "L'import de <strong>%label%</strong> est terminé"
+  list: Sélectionnez un connecteur
```
Comment (grena) [inline src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml]: @jjanvier told me that we don't do french translation, we let this job for nono

Comment (nidup) [inline src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml]: exact

File: src/Pim/Bundle/ImportExportBundle/Form/Type/JobInstanceType.php [hunk 2]
```diff
@@ -28,14 +29,19 @@ class JobInstanceType extends AbstractType
...
+
     /**
      * Constructor
      *
-     * @param ConnectorRegistry $connectorRegistry
+     * @param ConnectorRegistry   $connectorRegistry
+     * @param TranslatorInterface $translator
      */
-    public function __construct(ConnectorRegistry $connectorRegistry)
+    public function __construct(ConnectorRegistry $connectorRegistry, TranslatorInterface $translator)
```
Comment (nidup) [inline src/Pim/Bundle/ImportExportBundle/Form/Type/JobInstanceType.php]: this is a BC break thank you to add it in the changelog

Comment (solivier) [inline src/Pim/Bundle/ImportExportBundle/Form/Type/JobInstanceType.php]: done

**Pull request merged**